bootstrap/eks/config/manager/manager.yaml

```diff
@@ -25,5 +25,18 @@ spec:
         name: manager
       terminationGracePeriodSeconds: 10
       tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/control-plane
```

Comment:
I was also thinking we should make this key configurable like this:

```
key: ${K8S_CP_LABEL:=node-role.kubernetes.io/control-plane} 
```

so that when using EKS you can taint nodes to run "system" services.